INFO     logger:yaml_util.py:24 缓存数据读取成功{'token': 'a46328f58a48cfb5b844798a2c6d59e5'}
ERROR    logger:requests_util.py:34 【绑定邮箱--当前账号已绑定邮箱】接口请求异常，响应体：{'status': 505, 'msg': '验证换绑邮箱超时，请重试', 'data': []}
INFO     logger:requests_util.py:62 【绑定邮箱--当前账号已绑定邮箱】接口，断言：status=505，断言成功